pkg/ui/console/client.go

```diff
@@ -105,7 +106,8 @@ func (c Client) Fetch(ctx context.Context, failOnError bool) error {
 	request := client.FetchRequest{
 		Providers:         c.cfg.Providers,
 		UpdateCallback:    fetchCallback,
-		DisableDataDelete: viper.GetBool("disable-delete"),
+		DisableDataDelete: true, // TODO: use bool + also if history block is enable always turn into true
```

Comment:
It's already like this in Fetch code, maybe remove TODO? Rename field to `PreferDisableDataDelete`? Add comment? add `c.HistoryCfg != nil` here as well?

Reply:
Missed this one good catch.

Reply:
Considering removing disable delete to be true by default.